src/template/js/sdk/Transport/mock.js

```diff
@@ -83,7 +83,7 @@ function event(module, event, value) {
   if (id) {
     let message = JSON.stringify({
       jsonrpc: '2.0',
-      id: id,
+      id: listener[0],
```

Comment:
I'm confused by this line. Isn't `listener` a reference to a function, via `setListener`

Reply:
merge conflict resolution issue, fixed it.